openlibrary/core/admin.py

```diff
@@ -154,20 +154,65 @@ def _get_count_docs(ndays):
     return [d for d in docs if d]
 
 
-def get_stats(ndays=30):
+def get_stats(ndays=30, dev_prod=False):
     """Returns the stats for the past `ndays`"""
-    docs = _get_count_docs(ndays)
-    return {
-        'human_edits': Stats(docs, "human_edits", "human_edits"),
-        'bot_edits': Stats(docs, "bot_edits", "bot_edits"),
-        'lists': Stats(docs, "lists", "total_lists"),
-        'visitors': VisitorStats(docs, "visitors", "visitors"),
-        'loans': LoanStats(docs, "loans", "loans"),
-        'members': Stats(docs, "members", "total_members"),
-        'works': Stats(docs, "works", "total_works"),
-        'editions': Stats(docs, "editions", "total_editions"),
-        'ebooks': Stats(docs, "ebooks", "total_ebooks"),
-        'covers': Stats(docs, "covers", "total_covers"),
-        'authors': Stats(docs, "authors", "total_authors"),
-        'subjects': Stats(docs, "subjects", "total_subjects"),
-    }
+    if dev_prod:
+        docs = mock_get_stats()
+        return {
+            'human_edits': Stats(docs, "human_edits", "human_edits"),
+            'bot_edits': Stats(docs, "bot_edits", "bot_edits"),
+            'lists': Stats(docs, "lists", "total_lists"),
+            'visitors': Stats(docs, "visitors", "visitors"),
+            'loans': Stats(docs, "loans", "loans"),
+            'members': Stats(docs, "members", "total_members"),
+            'works': Stats(docs, "works", "total_works"),
+            'editions': Stats(docs, "editions", "total_editions"),
+            'ebooks': Stats(docs, "ebooks", "total_ebooks"),
+            'covers': Stats(docs, "covers", "total_covers"),
+            'authors': Stats(docs, "authors", "total_authors"),
+            'subjects': Stats(docs, "subjects", "total_subjects"),
+        }
+    else:
+        docs = _get_count_docs(ndays)
+        return {
+            'human_edits': Stats(docs, "human_edits", "human_edits"),
+            'bot_edits': Stats(docs, "bot_edits", "bot_edits"),
+            'lists': Stats(docs, "lists", "total_lists"),
+            'visitors': VisitorStats(docs, "visitors", "visitors"),
+            'loans': LoanStats(docs, "loans", "loans"),
+            'members': Stats(docs, "members", "total_members"),
+            'works': Stats(docs, "works", "total_works"),
+            'editions': Stats(docs, "editions", "total_editions"),
+            'ebooks': Stats(docs, "ebooks", "total_ebooks"),
+            'covers': Stats(docs, "covers", "total_covers"),
+            'authors': Stats(docs, "authors", "total_authors"),
+            'subjects': Stats(docs, "subjects", "total_subjects"),
+        }
```

Comment:
```suggestion
    docs = mock_get_stats() if use_mock_data else _get_count_docs(ndays)
    return {
        'human_edits': Stats(docs, "human_edits", "human_edits"),
        'bot_edits': Stats(docs, "bot_edits", "bot_edits"),
        'lists': Stats(docs, "lists", "total_lists"),
        'visitors': Stats(docs, "visitors", "visitors"),
        'loans': Stats(docs, "loans", "loans"),
        'members': Stats(docs, "members", "total_members"),
        'works': Stats(docs, "works", "total_works"),
        'editions': Stats(docs, "editions", "total_editions"),
        'ebooks': Stats(docs, "ebooks", "total_ebooks"),
        'covers': Stats(docs, "covers", "total_covers"),
        'authors': Stats(docs, "authors", "total_authors"),
        'subjects': Stats(docs, "subjects", "total_subjects"),
    }
```
DRY this function up.

Reply:
Should both the VisitorStats and LoanStats classes be deleted from ```openlibrary/core/admin.py``` or is this a temporary change?

Reply:
You're right, thanks for pointing this out.  How about this:

1. Update `mock_get_stats` to return the completed stats `dict`
2. Update `get_stats` to return `mock_get_stats()` if this is a dev instance.  Otherwise, return the actual stats (just as we've been doing).